src/sandstorm/run-bundle.c++

```diff
@@ -647,13 +647,13 @@ public:
     // Make sure the pidfile is close-on-exec.
     KJ_SYSCALL(fcntl(pidfile, F_SETFD, FD_CLOEXEC));
 
-    changeToInstallDir();
+    changeToInstallDirAndCheckAccess();
```

Comment:
I'm somewhat terrified that there could be some significant number of installations out there that randomly don't have /var/sandstorm writable by the group, but have the appropriate subdirectories writable such that things do actually still work. If we add a check in `continue_` then all those servers will die when they update. Similarly if the check is in `start` then they'll all die on their next reboot. I really don't want people to start fearing that an auto-update might randomly kill their server, so I'd rather not perform the check in these cases.